english/memories/ground-hog/kitchen/garden/beans.md

```diff
@@ -0,0 +1,5 @@
+Ah! Good choice. Beans are high in protein and have virtually no fat. They also have more fiber than most whole grain foods.
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.